packages/node_modules/@ciscospark/react-component-activity-post/src/index.js

```diff
@@ -28,13 +30,14 @@ function ActivityPost(props) {
     displayName,
     renderAdaptiveCard,
     cards,
-    sdkInstance
+    sdkInstance,
+    activityId
   } = props;
 
   return (
     <ActivityItemBase {...props}>
       {renderAdaptiveCard
-            ? <AdaptiveCard cards={cards} displayName={displayName} sdkInstance={sdkInstance} />
+            ? <AdaptiveCard cards={cards} displayName={displayName} sdkInstance={sdkInstance} activityId={activityId} />
```

Comment:
If you could, please convert this to a multi-line statement like so:
```js
<AdaptiveCard
  cards={cards}
  displayName={displayName}
  sdkInstance={sdkInstance}
  activityId={activityId}
/>
```
This will improve readabillity and can be simply amended to your last commit.